Gemfile

```diff
@@ -3,9 +3,6 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.2'
 
-gem 'rails-controller-testing'
-gem 'rspec-rails'
-
```

Comment:
- [ ] Kindly note that the requirements say that you should use respect for testing.

kindly make sure your development: test environment contains the RSpec gem in your gem file. example :point_down: 

```ruby
group :development, :test do
 ...
 gem 'rspec-rails'
end
```

- [ ] Then kindly write tests following the requirement with [`Rspec`](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controller_test_lesson.md)